### PR TITLE
#166816075 user notification opt-in/opt-out

### PR DESCRIPTION
**Pull request by devPinheiro**
#### What does this PR do?
- It contains  a feature for setting user preference on notifications

#### Description of Task to be completed?
- Update edit profile modal
- Add notification toggle 
- Write unit tests

#### How should this be manually tested?
- After cloning the repo, cd into the root folder

- Install all necessary dependencies by running 

  ``npm install``

- Launch the node server with

  ``npm run dev``

- Using a web browser visit the link below:

  ``localhost:8080/``

- To run local tests using Jest
  
  ``npm test``

#### Any background context you want to provide?
- N/A

Screenshot 2019-08-30 at 6 39 35 AM

#### What are the relevant pivotal tracker stories?
- PT Story link - `[#166816075] (https://www.pivotaltracker.com/story/show/166816075)`

### CODE REVIEW

**Review by fxola — APPROVED** [review]
LGTM

File: src/components/ProfileNavbar/index.spec.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ describe('<ProfileNavbar />', () => {
         </BrowserRouter>
       </Provider>
     );
-    const editButton = profileNav.find('.btn-navbar').simulate('click');
+    // const editButton = profileNav.find('.btn-navbar').simulate('click');
```
Comment (dharmykoya) [inline src/components/ProfileNavbar/index.spec.js]: please do remove this commented code

File: src/views/DraftArticle/DraftArticle.spec.js [hunk 2]
```diff
@@ -362,17 +362,17 @@ describe('PublishedArticle page', () => {
...
-        drafts: response.data,
-        fetchUnPublishedArticles: jest.fn(),
-        show: false,
-        auth: {
-          user: {
-            firstName: 'damilola'
-          }
-        }
-      };
+      // const props = {
```
Comment (dharmykoya) [inline src/views/DraftArticle/DraftArticle.spec.js]: remove this commented codes too... thanks

File: src/views/ReadArticlePage/readArticle.spec.js [hunk 3]
```diff
@@ -552,14 +552,13 @@ describe('ReadArticle Page', () => {
...
     const readArticlePage = readArticle.find('.article-title');
-    const instance = readArticle.instance();
-    const ratingCover = readArticle
-      .find('.article-body StarRatingComponent')
-      .simulate('click');
-    const rating = readArticle
-      .find('.article-body StarRatingComponent label')
-      .at(2)
-      .simulate('click');
+    // const instance = readArticle.instance();
```
Comment (dharmykoya) [inline src/views/ReadArticlePage/readArticle.spec.js]: remove this as well

**Review by dharmykoya — APPROVED** [review]
asides the commented codes.... the PR looks good to me

**Pull request merged**